contracts/RabbitHoleReceipt.sol

```diff
@@ -161,7 +162,7 @@ contract RabbitHoleReceipt is
         require(_exists(tokenId_), 'ERC721URIStorage: URI query for nonexistent token');
         require(QuestFactoryContract != IQuestFactory(address(0)), 'QuestFactory not set');
 
-        string memory questId = questIdForTokenId[tokenId_];
+        string storage questId = questIdForTokenId[tokenId_];
```

Comment:
I'll need to read up on storage vs memory. I thought memory would make sense here cause `questId` is a temp variable that only exists inside this function and doesn't need to be stored anywhere